.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,13 +4,43 @@ There are a few simple things to check before submitting your pull request
 that can help with the review process. You should delete these items
 from your submission, but they are here to help bring them to your attention.
 -->
+# Pull Request
 
-## Issues
-<!-- Link to related issues. Use closing keywords when appropriate -->
+*Issue link(s)*:
 
-## Summary
+<!--
+  Add Related Issues / PRs for context. Eg:
+    Related to elastic/repo#999
+    Resolves #123
+  If there is no issue link, take extra care to write a clear summary and label the PR just as you would label an issue to give additional context to reviewers.
+-->
+
+## Summary - What I changed
+
+<!--
+  Summarize your PR. Animated gifs are 💯. Code snippets are ⚡️. Examples & screenshots are 🔥
+-->
+
+## How To Test
+
+<!--
+  Some examples of what you could include here are:
+  * Links to GitHub action results for CI test improvements
+  * Sample data before/after screenshots (or short videos showing how something works)
+  * Copy/pasted commands and output from the testing you did in your local terminal window
+  * If tests run in GitHub, you can 🪁or 🔱, respectively, to indicate tests will run in CI
+  * Query used in your stack to verify the change
+-->
+
+## Checklist
 
+<!-- Delete any items that are not applicable to this PR. -->
 
+- [ ] Added a label for the type of pr: `bug`, `enhancement`, `Rule: New`, `Rule: Deprecation`, `Rule: Promote`, or `Rule: Tuning` so guidelines can be generated
```

Comment:
We now have `Hunt: New` and `Hunt: Tuning` as well.
Maybe we make those entirely separate?

Reply:
In the next commit, I'll duplicate the Rule workflows for this.